Signal/Symbols.xcassets/group/group-resizable.imageset/Contents.json

```diff
@@ -1,16 +0,0 @@
-{
-  "images" : [
-    {
-      "filename" : "group.pdf",
-      "idiom" : "universal"
-    }
-  ],
-  "info" : {
-    "author" : "xcode",
-    "version" : 1
-  },
-  "properties" : {
-    "preserves-vector-representation" : true,
-    "template-rendering-intent" : "template"
-  }
```

Comment:
Importantly, this "Use Vector Data" setting was removed and is not present for the other asset. When these were added, presumably this is what distinguished the "-resizable" variants.

It may be possible for us to replace the non-resizable one with the resizable, because we should be able to enable this for all PDF assets, but we shouldn't implicitly disable this.